programs/acme-sh.json

```
@@ -0,0 +1,10 @@
+{
+    "files": [
+        {
+            "path": "$HOME/.acme.sh",
+            "movable": false,
```

```suggestion
            "movable": true,
```
Hi, first of all, thanks for your contribution.

If I understand correctly, the files can be moved, so movable should be set to true.